src/main/java/de/tum/in/www1/artemis/web/rest/SubmissionResource.java

```diff
@@ -104,8 +109,15 @@ public SubmissionResource(SubmissionRepository submissionRepository) {
      * @return the ResponseEntity with status 200 (OK)
      */
     @DeleteMapping("/submissions/{id}")
+    @PreAuthorize("hasAnyRole('INSTRUCTOR', 'ADMIN')")
```

Comment:
Same as above, do we need to check if the user has instructor permissions on the course of the submission.

Reply:
Done :)